### ISSUE TITLE
Is this repository still under maintenance?

### ISSUE THREAD

**Issue by axiangcoding** [gocolly/colly#728]
No new version has been released for two years
It's a pity that such an elegant project was abandoned

**Comment by asciimoo** [comment]
It's in a low-effort maintenance mode - I think. The API is quite stable and works for most of the use cases. But of course, we appreciate any help around the project and we are open to new contributions ;)

**Comment by axiangcoding** [comment]
@asciimoo Yes I also think the API is stable enough. But in the past two years, the update has not been released, making it feel that the repo has lost maintenance.

And I do have the idea of localizing the docs, but I see https://go-colly.org/ don't support multiple languages, so I will consider making an additional website sometime in the future to introduce this project

**Comment by SebTM** [comment]
@asciimoo I can understand, maybe you could do a release of the changes since 2020 and merge some of the fixes for the site/docs? If you don't have the time, I can offer some help here.

**Comment by philicious** [comment]
@asciimoo I would also like to kindly ask for release with latest fixes. especially the fix for the DOS vuln in xmlquery <1.3.1 would be great to have in a release

**Comment by RTodorov** [comment]
Hi @asciimoo, is there any chance you could generate a version from master? Thank you!

**Comment by kenliu** [comment]
+1

**Comment by cristaloleg** [comment]
Kindly pinging to cur a new release 🙂

**Comment by asciimoo** [comment]
I've created a new release with the current codebase and updated dependencies: https://github.com/gocolly/colly/releases/tag/v2.2.0 .
